### PR TITLE
Global nav

### PR DESCRIPTION
**Pull request by erikguntner**
This PR includes the following updates to the global nav bar:

- Adding the Settings, Help, and Logout items to the dropdown menu
- Creating the settings page
- Display user initials in Avatar

We aren't currently collecting the user's name so right now it uses a fake name.

### CODE REVIEW

**Comment by erikguntner** [comment]
https://github.com/hackforla/HomeUniteUs/assets/27253583/3dfb5c26-6b63-45d2-9779-364639b6ea0a

File: app/src/components/common/Header.tsx [hunk 1]
```diff
@@ -18,7 +18,14 @@ import {useAuth} from '../../app/hooks/useAuth';
...
 import {useAppDispatch} from '../../app/hooks/store';
 import {setCredentials} from '../../app/authSlice';
-import {useNavigate} from 'react-router-dom';
+import {Link, useNavigate} from 'react-router-dom';
+
+const userName = 'Hank Hill';
+
+function getInitials(name: string) {
+  const splitName = name.split(' ');
+  return splitName[0][0] + splitName[1][0];
```
Comment (paulespinosa) [inline app/src/components/common/Header.tsx:27]: This should have more checks. For example, check if `splitName` has more than 1 element.

**Review by paulespinosa — CHANGES_REQUESTED** [review]

File: app/src/components/common/Header.tsx (hunk 1 above)
Comment (erikguntner) [reply app/src/components/common/Header.tsx]: Thanks! Added a check for length and also to use the last element in case it's not just first and last name.

**Review by paulespinosa — APPROVED** [review]
Thank you

**Pull request merged**